service/src/main/java/bio/terra/externalcreds/pact/ProviderStates.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.externalcreds.pact;
+
+// Note: This is a regular class, not an Enum, because Java doesn't consider enum values to be
+// constant at compile time, which is a problem when trying to use these states in annotations in
+// TestVerification.java.
+
+public class ProviderStates {
+  public static final String USER_IS_REGISTERED = "test_user@test.com is registered with ECM";
```

Comment:
I think it is best practice for a value like this email to be a provider supplied state value so that we dont have to make sure that both the consumer tests and provider tests have the same hardcoded value. Then this state could be more generic like "a user is registered with ECM".